docs/topics/about-home-var.adoc

```diff
@@ -4,9 +4,9 @@
 
 :_content-type: CONCEPT
 [id="about-home-var_{context}"]
-= Use of `<MTA_HOME>` in this guide
+= Use of `<{ProductShortName}_HOME>` in this guide
 
-This guide uses the `<MTA_HOME>` replaceable variable to denote the path to your {ProductShortName} installation. The installation directory is the `{ProductDistribution}` directory where you extracted the {ProductShortName} `.zip` file.
+This guide uses the `<{ProductShortName}_HOME>` replaceable variable to denote the path to your {ProductShortName} installation. The installation directory is the `{ProductDistribution}` directory where you extracted the {ProductShortName} `.zip` file.
```

Comment:
The installation directory is the mta-cli-5.3.0.Final directory where you extracted the {ProductShortName} .zip file.

mta-cli-5.3.0 is being displayed in the Preview instead of mtr-cli